roscala/src/test/scala/coop/rchain/rosette/prim/TupleSpec.scala

```diff
@@ -22,16 +22,66 @@ class TupleSpec extends FlatSpec with Matchers {
     monitor = null
   )
 
+  /**
+   * tuple-cons
+   */
   "tplCons" should "correctly cons an Ob with a Tuple" in {
-    val test = Seq(RFixnum(2), RFixnum(3), RFixnum(4))
+    val tup = Seq(RFixnum(2), RFixnum(3), RFixnum(4))
     val newCtxt =
-      ctxt.copy(nargs = 2, argvec = Tuple.cons(RFixnum(1), Tuple(Tuple(test))))
-    tplCons.fn(newCtxt) should be(Right(Tuple.cons(RFixnum(1), Tuple(test))))
+      ctxt.copy(nargs = 2, argvec = Tuple.cons(RFixnum(1), Tuple(Tuple(tup))))
+    tplCons.fn(newCtxt) should be(Right(Tuple.cons(RFixnum(1), Tuple(tup))))
   }
 
   it should "fail for non-tuple arguments" in {
     val newCtxt = ctxt.copy(nargs = 5, argvec = Tuple(5, Ob.NIV))
     tplCons.fn(newCtxt) should be('left)
   }
 
+  /**
```

Comment:
Sorry, should have mentioned that earlier.
One-line outer object/class comments should look like this: `/** tuple-cons */`